src/markdown/ciceromark@0.3.0.cto

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace org.accordproject.ciceromark
+
+import org.accordproject.commonmark.Node from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+import org.accordproject.commonmark.Child from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+
+/**
+ * A model for Accord Project extensions to commonmark
+ */
+
+abstract concept Element extends Child {
+  o String name
+  o String elementType optional
+}
+
+concept Variable extends Element {
+  o String value
+  o String identifiedBy optional
```

Comment:
Doesn't have to be in this PR, but I'd love to be able to access Concerto decorator values from the Variable node. That would provide an extensibility mechanism for people, in that they could add a decorator to their Concerto type, and the value of the decorator would flow all the into the CiceroMark Variable node, where it could be used to control rendering (for example).

Reply:
If you can provide a model for that we could easily add it as optional (with the caveat that those things tend to have to be adjusted when we actually hook them up). I'm happy for that change though.

Reply:
I'm not sure I would know where to hang on those annotations, to variables?

Reply:
They'd make sense on any element that is related to a Concerto type. So, perhaps `Element`? 

Something like:

```
import concerto.metamodel.Decorator from https://models.accordproject.org/concerto/metamodel.cto 
...
o Decorator[] decorators optional
```

It would be a backwards compatible change however - so don't feel compelled to roll it into this PR.

Reply:
yes maybe we can do that as a second pass. Also we do have some room for changes between now and when we roll out cicero vnext?